kubetest/anywhere.go

```diff
@@ -45,7 +45,7 @@ const kubernetesAnywhereConfigTemplate = `
 .phase1.gce.instance_type="n1-standard-1"
 .phase1.gce.project="{{.Project}}"
 .phase1.gce.region="us-central1"
-.phase1.gce.zone="us-central1-b"
+.phase1.gce.zone="us-central1-c"
```

Comment:
Fair question. I hadn't seen a need to support overriding it with a CLI flag to have it differ between jobs, but doing it in order to prevent having to rebuild the image is potentially worthwhile. I'll give it some thought.